great_expectations_provider/operators/great_expectations.py

```diff
@@ -247,11 +248,19 @@ def make_connection_configuration(self) -> Dict[str, str]:
             odbc_connector = ""
             if conn_type in ("redshift", "postgres"):
                 odbc_connector = "postgresql+psycopg2"
+                database_name = self.schema
             elif conn_type == "mysql":
                 odbc_connector = "mysql"
+                database_name = self.schema
             else:
```

Comment:
will `conn_type = "mssql"` or something similar be passed to this method? If so could we please try something like 

```python
elfi conn_type == "mssql":
    ...
else:
    raise ValueError(f"conn_type {conn_type} is not supported")
```